### PR TITLE
[TS] LPS-170303 Using 1.2.2.v20211118-0920-LIFERAY-CACHE.jar instead of 1.2.2.v20211119-2358-LIFERAY-CACHED.jar

### PR DESCRIPTION
**Pull request by evibudai**
Hi Team,

https://issues.liferay.com/browse/LPS-170303

Previous PR: https://github.com/liferay-core-infra/liferay-portal/pull/3023

I'm re-sending this PR, based on @yuhai 's comment:
https://github.com/liferay-core-infra/liferay-portal/pull/3023#issuecomment-1383639558
"...I think we may do the patch based on 1.2.2.v20211118-0920-LIFERAY-CACHED version. We need not to patch LPS-129450 if using this way. You may rename the version 1.2.2.v20211119-2358.LIFERAY-PATCHED-2 for 1.2.2.v20211118-0920-LIFERAY-PATCHED-1 in ../third-party/org-eclipse-equinox-http-servlet/bnd.bnd and change build version in build.gradle. ..."

@yuhai was kind enough to check the nexus repo, and find 1.2.2.v20211118-0920-LIFERAY-CACHE.jar, which contains not only the LPS I wanted to patch in the previous PR, but also one other missing LPS, and LPS-140601 (that was present in the previous base version 1.2.2.v20211119-2358-LIFERAY-CACHED).
Namely it contains: LPS-108112, LPS-129450 and LPS-140601

Since the patch file for LPS-163139 already exists, I only needed to change the base version from 1.2.2.v20211119-2358-LIFERAY-CACHED to 1.2.2.v20211118-0920-LIFERAY-CACHED.
In total:
- In org-eclipse-equinox-http-servlet/build.gradle to 1.2.2.v20211118-0920-LIFERAY-CACHED
- In org-eclipse-equinox-http-servlet/bnd.bnd to 1.2.2.v20211118-0920-LIFERAY-PATCHED-1
- In /portal-osgi-web-wab-extender/build.gradle to 1.2.2.v20211118-0920-LIFERAY-PATCHED-1

Please review my PR!

Thank you and best regards,
Évi

### CODE REVIEW

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by yuhai** [comment]
@evibudai, the fix looks good to me. But the commit https://github.com/liferay-core-infra/liferay-portal/pull/3070/commits/a8ef8e95648ea8d47bae2f7e32285901ac7f4f8f message is incorrect. Atually, **1.2.2.v20211119-2358-LIFERAY-CACHED** included LPS-140601. It didn't include LPS-108112 and LPS-129450. Could you please change the commit message and force it? After git force, need not to execute ci test since only change the commit message.

**Comment by evibudai** [comment]
>  Atually, 1.2.2.v20211119-2358-LIFERAY-CACHED included LPS-140601. It didn't include LPS-108112 and LPS-129450. 

Yes, I know that, the way I phrased the commit means exactly this.

> 1.2.2.v20211119-2358-LIFERAY-CACHED.jar that did not contain all three LPS

@yuhai   "not contain all three LPS" means that it did not contain all three of them, only one (LPS-140601)

I detailed this here too: 
"....2.2.v20211118-0920-LIFERAY-CACHE.jar, which contains not only the LPS I wanted to patch in the previous PR (LPS-129450), but also one other missing LPS (LPS-108112), and LPS-140601 (that was present in the previous base version 1.2.2.v20211119-2358-LIFERAY-CACHED)."

But if it is ambiguous, I can change the message.
Does this seem good?:
"LPS-170303 Using 1.2.2.v20211118-0920-LIFERAY-CACHE.jar that contains LPS-108112, LPS-129450 and LPS-140601, instead of 1.2.2.v20211119-2358-LIFERAY-CACHED.jar that only contains LPS-140601"

**Comment by yuhai** [comment]
> > Atually, 1.2.2.v20211119-2358-LIFERAY-CACHED included LPS-140601. It didn't include LPS-108112 and LPS-129450.
> 
> Yes, I know that, the way I phrased the commit means exactly this.
> 
> > 1.2.2.v20211119-2358-LIFERAY-CACHED.jar that did not contain all three LPS
> 
> @yuhai "not contain all three LPS" means that it did not contain all three of them, only one (LPS-140601)
> 
> I detailed this here too: "....2.2.v20211118-0920-LIFERAY-CACHE.jar, which contains not only the LPS I wanted to patch in the previous PR (LPS-129450), but also one other missing LPS (LPS-108112), and LPS-140601 (that was present in the previous base version 1.2.2.v20211119-2358-LIFERAY-CACHED)."
> 
> But if it is ambiguous, I can change the message. Does this seem good?: "LPS-170303 Using 1.2.2.v20211118-0920-LIFERAY-CACHE.jar that contains LPS-108112, LPS-129450 and LPS-140601, instead of 1.2.2.v20211119-2358-LIFERAY-CACHED.jar that only contains LPS-140601"

@evibudai, sorry, I misunderstand this. Your current commit message is correct. If you may update the commit message by using you mentioned as the below, it will be very clearly to me. Thanks.

> "LPS-170303 Using 1.2.2.v20211118-0920-LIFERAY-CACHE.jar that contains LPS-108112, LPS-129450 and LPS-140601, instead of 1.2.2.v20211119-2358-LIFERAY-CACHED.jar that only contains LPS-140601"

**Review by yuhai — APPROVED** [review]

**Comment by yuhai** [comment]
@dantewang, the fix looks good to me.

**Comment by evibudai** [comment]
@yuhai all right

File: modules/third-party/org-eclipse-equinox-http-servlet/bnd.bnd [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Bundle-SymbolicName: org.eclipse.equinox.http.servlet
-Bundle-Version: 1.2.2.v20211119-2358.LIFERAY-PATCHED-2
\ No newline at end of file
+Bundle-Version: 1.2.2.v20211118-0920-LIFERAY-PATCHED-1
```
Comment (yuhai) [inline modules/third-party/org-eclipse-equinox-http-servlet/bnd.bnd:2]: @evibudai, I just noticed the version is "-" to append the -LIFERAY-PATCHED-1, others used ".". For example, ../third-party/biz-aQute-bnd-annotation/bnd.bnd.  Please changed "1.2.2.v20211118-0920-LIFERAY-PATCHED-1" for 1.2.2.v20211118-0920.LIFERAY-PATCHED-1.

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:sf

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@zhangyan0701 A frontend building issue is blocking running of ci-test-relevant?

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
> @zhangyan0701 A frontend building issue is blocking running of ci-test-relevant?

I haven't heard of a frontend building issue. And I didn't reproduce it locally. I re-trigger relevant test, see how it goes this time.

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/129023

**Pull request closed without merging**